README.md

```diff
@@ -117,4 +117,6 @@ Feedstock Maintainers
 =====================
 
 * [@ocefpaf](https://github.com/ocefpaf/)
+* [@pedrovma](https://github.com/pedrovma)
+* [@jGaboardi](https://github.com/jGaboardi)
```

Comment:
The README is Auto generated, sorry for not mentioning this before. You need to add your GH handles in the meta.yaml.

Reply:
LOL. OK. I should have know that...

Reply:
Does this mean I should increase the build number in `meta.yml`?

Reply:
No need, this is more for the infrastructure, it will give you commit rights.